docs/platform/release-notes/tizen-6-0-m1.md

```diff
@@ -0,0 +1,692 @@
+# ﻿Tizen 6.0 Public M1 Release Notes
+
+Release Date: May 31, 2020

```

Comment:
To maintain consistency with other release notes, please change "Date" to "date":

Release date: May 31, 2020

Reference: https://developer.tizen.org/tizen/release-notes/tizen-5.5-public-m1

Reply:
Release date : Tizen 4.0 Public M3, Tizen 5.0 Public M1 ~ M2, Tizen 5.5 Public M1
Release Date : Tizen 3.0 Public M2 ~ M4, Tizen 4.0 Public M1 ~ M2, Tizen 5.5 M2
Actually consistency has not been met. Please recommend which one is better for Tizen 6.0 M1 and later Release Notes. (date or Date)

Reply:
will use "date" instead of "Date" for consistency from this release.

Reply:
@wtprince, 

Thank you for accepting. Also, after an internal discussion, we have come to a conclusion to use lower case going forward.

Release date: May 31, 2020